.gitignore

```diff
@@ -30,3 +30,4 @@ browsers.processed.js
 tasks/selenium
 bundle.js
 bundle.js.map
+package-lock.json
```

Comment:
We don't want to ignore package lock for when we eventually get to npm 5.

Reply:
nah, it's fine. i keep meaning to gitignore it or npmrc ignore it and I keep wanting to avoid a build that'll run all the tests.